certified-connectors/RescoReports/apiProperties.json

```diff
@@ -0,0 +1,71 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
```

Comment:
@JuroJ can you please confirm if your connector is using Azure Active Directory as an identity provider?